static/app/components/events/eventStatisticalDetector/lineChart.tsx

```diff
@@ -4,24 +4,34 @@ import {useTheme} from '@emotion/react';
 import ChartZoom from 'sentry/components/charts/chartZoom';
 import {LineChart as EChartsLineChart} from 'sentry/components/charts/lineChart';
 import getBreakpointChartOptionsFromData from 'sentry/components/events/eventStatisticalDetector/breakpointChartOptions';
-import type {PageFilters} from 'sentry/types';
+import type {EventsStatsData, PageFilters} from 'sentry/types';
 import type {Series} from 'sentry/types/echarts';
 import useRouter from 'sentry/utils/useRouter';
 import type {NormalizedTrendsTransaction} from 'sentry/views/performance/trends/types';
 
 interface ChartProps {
   datetime: PageFilters['datetime'];
   evidenceData: NormalizedTrendsTransaction;
-  percentileSeries: Series[];
+  // TODO @athena: Refactor functionBreakpointChart to use percentileData
```

Comment:
For now I keep both the option to pass the series here and raw data. Later when I'm adding chartcuterie for function regression endpoint I'll refactor to only pass raw data + info on how to transform it to series.